doc/cookbook/gpus.md

```diff
@@ -0,0 +1,226 @@
+# Utilizing GPUs
+
+Pachyderm has alpha support for utilizing GPUs within Pachyderm pipelines (e.g., for training machine learning models).  To do this you will need to:
+
+1) [Create a Docker image that is able to utilize GPUs](gpus.html#creating-a-gpu-enabled-docker-image)
+2) [Write a pipeline spec that specifies GPU nodes](gpus.html#writing-your-pipeline-specification)
+3) [Deploy a GPU enabled pachyderm cluster](gpus.html#deploy-a-gpu-enabled-pachyderm-cluster)
+
+For a concrete example, see our [example Tensorflow pipeline](https://github.com/pachyderm/pachyderm/tree/master/doc/examples/ml/tensorflow) for image-to-image translation, which includes a pipeline specification for running model training on a GPU node.
+
+## Creating a GPU Enabled Docker Image
+
+For your Docker image, you'll want to use or build an image that can utilize GPU resources.  If you are using Tensorflow, for example, you could build your Docker image `FROM` the public GPU enabled Tensorflow image:
+
+```
+FROM tensorflow/tensorflow:0.12.0-gpu
+...
+```
+
+Or you might follow [this general guide](https://github.com/NVIDIA/nvidia-docker) for working with Docker and NVIDIA.
```

Comment:
Yeah, we can mention that it is untested.  According to this it should work: http://blog.clarifai.com/how-to-scale-your-gpu-cloud-infrastructure-with-kubernetes/#.WUmHiXUrLkx